src/link/script.y

```diff
@@ -388,6 +392,20 @@ static void setActiveTypeAndIdx(SectionType type, uint32_t idx) {
 	}
 }
 
+static void setFloatingSectionType(SectionType type) {
+	if (nbbanks(type) == 1) {
+		setActiveTypeAndIdx(type, 0); // There is only a single bank anyway, so just set the index to 0.
```

Comment:
Should this give a warning, since `ROM0 FLOATING` or `WRAM0 FLOATING` are redundant and may indicate a mistake?

Reply:
OTOH it could make sense if the script is auto-generated, which seems plausible for toolchains. And RGBLINK doesn't have a warning flag system, so... eh?